Makefile.custom.mk

```diff
@@ -8,13 +8,13 @@ vet: ## Run go vet against code.
 
 .PHONY: test-unit
 test-unit: ginkgo generate fmt vet envtest ## Run unit tests
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" $(GINKGO) -p --nodes 4 -r -randomize-all --randomize-suites --skip-package=tests --cover --coverpkg=`go list ./... | grep -v fakes | tr '\n' ','` ./...
+	UPDATE_GOLDEN_FILES=true KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" $(GINKGO) -p --nodes 4 -r -randomize-all --randomize-suites --skip-package=tests --cover --coverpkg=`go list ./... | grep -v fakes | tr '\n' ','` ./...
```

Comment:
Why setting `UPDATE_GOLDEN_FILES=true` this should only be used when one want to actually update golden files no on every test run.

Reply:
Well I did not want to add a custom test and having one make test makes it easier for people to use

Reply:
This breaks the purpose of testing, golden files should not be updated here